main.js

```diff
@@ -68,6 +68,7 @@ function addPassword(service, password) {
   copyButton.type = 'button';
   copyButton.className = 'btn btn-outline-primary';
   copyButton.innerHTML = '<img src="clipboard.svg" alt="">';
+  copyButton.id = 'copyButton';
```

Comment:
ID values are required to be unique. This is your problem.

Reply:
Here let me PR your branch with my solution

Reply:
Oh my gosh. Duh.